covid_moonshot/app.py

```diff
@@ -25,32 +24,32 @@ def analyze_runs_cli(
     Parameters
     ----------
     run_details_json_file : str
-        json file containing run metadata. The file should contain a
-        json object with values deserializable to `RunDetails`
+        JSON file containing run metadata. The file should contain a
+        JSON object with values deserializable to `RunDetails`
     complex_project_path : str
-        path to the FAH project directory containing configuration for
+        Path to the FAH project directory containing configuration for
         simulations of the complex,
         e.g. '/home/server/server2/projects/13422'
     complex_project_data_path : str
-        path to the FAH project data directory containing output data
+        Path to the FAH project data directory containing output data
         from simulations of the complex,
         e.g. "/home/server/server2/data/SVR314342810/PROJ13422"
```

Comment:
I think we want to refactor things so that the project paths are reconstructed from the `/home/server/server2/data/SVR314342810` server path + the run_details path in a future iteration.